fl4health/parameter_exchange/sparse_coo_parameter_exchanger.py

```diff
@@ -0,0 +1,154 @@
+import math
+from logging import INFO
+from typing import Callable, Dict, List, Optional, Tuple
+
+import torch
+import torch.nn as nn
+from flwr.common.logger import log
+from flwr.common.typing import Config, NDArrays
+from torch import Tensor
+from torch.nn.modules import Module
+
+from fl4health.parameter_exchange.parameter_packer import SparseCooParameterPacker
+from fl4health.parameter_exchange.partial_parameter_exchanger import PartialParameterExchanger
+
+
+class SparseCooParameterExchanger(PartialParameterExchanger[Tuple[NDArrays, NDArrays, List[str]]]):
+    def __init__(self, sparsity_level: float, score_gen_function: Callable[..., Dict[str, Tensor]]) -> None:
```

Comment:
Unless I'm mistaken, I think we can be more explicit in the Callable signature here, as all `score_gen_functions` should be `Callable[[nn.Module, nn.Module], Dict[str, Tensor]]`

Reply:
Yes, good point!